### PR TITLE
Gomplate to use predefined config

### PR DESCRIPTION
**Pull request by max-lobur**
## what

- Define gomplate config and pass it throught CLI flag to every gomplate invocation
- Passing as the very last option to avoid injecting override in the env vars dereferenced later in same command
- unset GOMPLATE_CONFIG is not required as CLI option always takes precedence over env var, I tested.

### CODE REVIEW

**Comment by max-lobur** [comment]
Testing in few repos ATM

File: modules/readme/Makefile [hunk 1]
```diff
@@ -29,8 +29,7 @@ readme/lint:
 
 ## Create README.md by building it from README.yaml
 readme/build: readme/deps $(README_DEPS)
-	@gomplate --file $(README_TEMPLATE_FILE) \
-		--out $(README_FILE) 
+	@gomplate --file $(README_TEMPLATE_FILE) --out $(README_FILE) --config $(BUILD_HARNESS_PATH)/configs/gomplate.yaml
```
Comment (osterman) [inline modules/readme/Makefile:32]: Does `--config` override `GOMPLATE_CONFIG`?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -16,3 +16,4 @@
 # Cloud Posse must review any changes to GitHub templates actions
 templates/.github/*             @cloudposse/engineering
 templates/terraform/.github/*   @cloudposse/engineering
+**/.gomplate.*
```
Comment (osterman) [inline .github/CODEOWNERS]:
```suggestion
**/.gomplate.* @cloudposse/admins
**/configs/gomplate.yaml @cloudposse/admins
```

File: modules/readme/Makefile (hunk 1 above)
Comment (max-lobur) [reply modules/readme/Makefile:32]: yes, tested this

**Review by osterman — APPROVED** [review]

**Pull request merged**